### PR TITLE
fixed lenght undefined error

### PR DESCRIPTION
**Pull request by maheshsarade001**

### CODE REVIEW

File: client/src/components/admin/AdminManagePlans.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const AdminManagePlans = () => {
   const allPlans = useSelector((state) => state.plans.allPlans);
   const updateLoader = useSelector((state) => state.plans.updateLoader);
   const loading = useSelector((state) => state.plans.loading);
-  console.log(loading, "state");
+  console.log(allPlans, "state");
```
Comment (shivamvijaywargi) [inline client/src/components/admin/AdminManagePlans.jsx]: Console.log is not required.

File: client/src/components/admin/AdminManagePlans.jsx [hunk 2]
```diff
@@ -242,7 +239,7 @@ const AdminManagePlans = () => {
                   >
                     <td className="text-center">{index + 1}</td>
                     <td className="text-center">{plan.planName}</td>
-                    <td className="text-center">₹ {plan.amount}</td>
+                    <td className="text-center">{plan.amount}</td>
```
Comment (shivamvijaywargi) [inline client/src/components/admin/AdminManagePlans.jsx]: May I know why did you remove this?

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please resolve the comments

**Review by sub1120 — APPROVED** [review]

File: client/src/components/admin/AdminManagePlans.jsx (hunk 2 above)
Comment (sub1120) [reply client/src/components/admin/AdminManagePlans.jsx]: will do it. thanks!

File: client/src/components/admin/AdminManagePlans.jsx (hunk 1 above)
Comment (sub1120) [reply client/src/components/admin/AdminManagePlans.jsx]: will do it. thanks!

**Review by sub1120 — APPROVED** [review]

File: client/src/components/admin/AdminManagePlans.jsx (hunk 2 above)
Comment (shivamvijaywargi) [reply client/src/components/admin/AdminManagePlans.jsx]: Why did you merge without resolving them?

File: client/src/components/admin/AdminManagePlans.jsx (hunk 1 above)
Comment (shivamvijaywargi) [reply client/src/components/admin/AdminManagePlans.jsx]: Why did you merge without resolving them?
